doc/nrf/working_with_nrf/nrf53/thingy53.rst

```diff
@@ -220,23 +220,14 @@ The overlay is applied automatically.
 MCUboot bootloader
 ==================
 
-Each sample must enable the MCUboot bootloader and use the same configuration of the bootloader as all of the Nordic Thingy:53-compatible samples in the |NCS|.
+MCUboot bootloader is enabled by default for Thingy:53 in board Kconfig file.
```

Comment:
> I wonder if we should inform that these feature is only available in the NCS (this would not be enabled out-of-the-box if we use vanilla Zephyr).

To be honest, I am not really convinced about that. These docs relates to only NCS and for Vanilla Zephyr there are other docs. For now i did not addressed this comment in code.